specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/schema/http.schema.patch.json

```diff
@@ -237,6 +237,114 @@
+                                                {
+                                                    "type": "object",
+                                                    "patternProperties":
+                                                    {
+                                                        "^[a-zA-Z]+[a-zA-Z0-9\\._\\-]*$":
+                                                        {
+                                                            "$ref": "#/$defs/validator/type"
+                                                        }
+                                                    }
+                                                },
+                                                "query":
+                                                {
+                                                    "type": "object",
+                                                    "patternProperties":
+                                                    {
+                                                        "^[a-zA-Z]+[a-zA-Z0-9\\._\\-]*$":
+                                                        {
+                                                            "$ref": "#/$defs/validator/type"
+                                                        }
+                                                    }
+                                                }
+                                            },
+                                            "additionalProperties": false
+                                        },
+                                        "content":
+                                        {
+                                            "$ref": "#/$defs/validator/type"
+                                        }
+                                    },
+                                    "anyOf":
+                                    [
+                                        {
+                                            "required":
+                                            [
+                                                "path",
+                                                "headers"
+                                            ]
+                                        },
+                                        {
+                                            "required":
+                                            [
+                                                "path",
+                                                "params"
+                                            ]
+                                        },
+                                        {
+                                            "required":
+                                            [
+                                                "path",
+                                                "content"
+                                            ]
+                                        }
+                                    ],
```

Comment:
Looks like `path` is just always required, right?

Reply:
I meant to express **path and (headers or params or content)** is required. We can simplify this to path is required ➡️ in that case people will be able to specify just a **path**, I guess it will not cause any issues, it will just be a no-op. Pls LMK.